### PR TITLE
CambridgeUniversityPress.CambridgeReader version 1.0210.1

### PR DESCRIPTION
**Pull request by solomoncyj**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/40067)

### CODE REVIEW

File: manifests/c/CambridgeUniversityPress/CambridgeReader/1.0210.1/CambridgeUniversityPress.CambridgeReader.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: CambridgeUniversityPress.CambridgeReader
+PackageVersion: 1.0210.1
```
Comment (ItzLevvie) [inline manifests/c/CambridgeUniversityPress/CambridgeReader/1.0210.1/CambridgeUniversityPress.CambridgeReader.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.0
```

File: manifests/c/CambridgeUniversityPress/CambridgeReader/1.0210.1/CambridgeUniversityPress.CambridgeReader.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: CambridgeUniversityPress.CambridgeReader
+PackageVersion: 1.0210.1
```
Comment (ItzLevvie) [inline manifests/c/CambridgeUniversityPress/CambridgeReader/1.0210.1/CambridgeUniversityPress.CambridgeReader.installer.yaml:5]:
```suggestion
PackageVersion: 1.0
```

File: manifests/c/CambridgeUniversityPress/CambridgeReader/1.0210.1/CambridgeUniversityPress.CambridgeReader.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: CambridgeUniversityPress.CambridgeReader
+PackageVersion: 1.0210.1
```
Comment (ItzLevvie) [inline manifests/c/CambridgeUniversityPress/CambridgeReader/1.0210.1/CambridgeUniversityPress.CambridgeReader.yaml:5]:
```suggestion
PackageVersion: 1.0
```

Comment (ItzLevvie) [reply manifests/c/CambridgeUniversityPress/CambridgeReader/1.0210.1/CambridgeUniversityPress.CambridgeReader.yaml:5]: This needs to match Control Panel so `winget upgrade CambridgeUniversityPress.CambridgeReader` doesn't end up in a upgrade loop for version mismatch.

File: manifests/c/CambridgeUniversityPress/CambridgeReader/1.0210.1/CambridgeUniversityPress.CambridgeReader.locale.en-US.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/c/CambridgeUniversityPress/CambridgeReader/1.0210.1/CambridgeUniversityPress.CambridgeReader.locale.en-US.yaml:5]: This needs to match Control Panel so `winget upgrade CambridgeUniversityPress.CambridgeReader` doesn't end up in a upgrade loop for version mismatch.

File: manifests/c/CambridgeUniversityPress/CambridgeReader/1.0210.1/CambridgeUniversityPress.CambridgeReader.installer.yaml (hunk 2 above)
Comment (ItzLevvie) [reply manifests/c/CambridgeUniversityPress/CambridgeReader/1.0210.1/CambridgeUniversityPress.CambridgeReader.installer.yaml:5]: This needs to match Control Panel so `winget upgrade CambridgeUniversityPress.CambridgeReader` doesn't end up in a upgrade loop for version mismatch.

Comment (ItzLevvie) [reply manifests/c/CambridgeUniversityPress/CambridgeReader/1.0210.1/CambridgeUniversityPress.CambridgeReader.installer.yaml:5]: You will also need to update the version folder as well.

Change `manifests/c/CambridgeUniversityPress/CambridgeReader/1.0210.1` to `manifests/c/CambridgeUniversityPress/CambridgeReader/1.0`

File: manifests/c/CambridgeUniversityPress/CambridgeReader/1.0210.1/CambridgeUniversityPress.CambridgeReader.locale.en-US.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/c/CambridgeUniversityPress/CambridgeReader/1.0210.1/CambridgeUniversityPress.CambridgeReader.locale.en-US.yaml:5]: You will also need to update the version folder as well.

Change `manifests/c/CambridgeUniversityPress/CambridgeReader/1.0210.1` to `manifests/c/CambridgeUniversityPress/CambridgeReader/1.0`

File: manifests/c/CambridgeUniversityPress/CambridgeReader/1.0210.1/CambridgeUniversityPress.CambridgeReader.yaml (hunk 3 above)
Comment (ItzLevvie) [reply manifests/c/CambridgeUniversityPress/CambridgeReader/1.0210.1/CambridgeUniversityPress.CambridgeReader.yaml:5]: You will also need to update the version folder as well.

Change `manifests/c/CambridgeUniversityPress/CambridgeReader/1.0210.1` to `manifests/c/CambridgeUniversityPress/CambridgeReader/1.0`

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/c/CambridgeUniversityPress/CambridgeReader/1.0210.1/CambridgeUniversityPress.CambridgeReader.locale.en-US.yaml (hunk 1 above)
Comment (solomoncyj) [reply manifests/c/CambridgeUniversityPress/CambridgeReader/1.0210.1/CambridgeUniversityPress.CambridgeReader.locale.en-US.yaml:5]: image

**Comment by solomoncyj** [comment]
image

**Comment by solomoncyj** [comment]
https://github.com/microsoft/winget-pkgs/pull/40066

**Pull request closed without merging**